MANIFEST.in

```diff
@@ -13,4 +13,9 @@ recursive-include jujugui/templates *
 recursive-exclude jujugui/static/gui/src *
 recursive-exclude jujugui/tests *
 recursive-exclude test *
+recursive-exclude jujugui/static/gui/build/app/components test-*.js
 
+# To get only the *min.js files we first remove all js files and then add the
+# min.js files back in
+recursive-exclude jujugui/static/gui/build *.js
+recursive-include jujugui/static/gui/build *min.js
```

Comment:
Would this reinclude the minified test files? I am sure it wouldn't just worth checking?